charts/trino/values.yaml

```diff
@@ -402,6 +402,15 @@ catalogs:
 #  memory: |
 #    connector.name=memory
 #    memory.max-data-per-node=128MB
+#  postgresql: |
+#    connector.name=postgresql
+#    connection-url=jdbc:postgresql://postgres-service.my-postgres.svc:5432/my_db
+#    connection-user=${ENV:POSTGRES_USER}
```

Comment:
The description before the example could also mention that sensitive values like credentials should be referenced from envrionment variables, mounted from a secret (using the `envFrom` section), and reference https://trino.io/docs/current/security/secrets.html